nextjs/src/app/[locale]/theme/page.tsx

```diff
@@ -46,16 +54,27 @@ export default function Theme() {
         <Text markdown={intro.text} className="grid gap-8" />
       </Intro>
 
-      <Container padding={"start-padding"} id="solutions" background="neutral">
+      <Container id="Solutions" padding={"start-padding"} background="neutral">
         <CardSlider
           title={solutions.title}
           description={solutions.description}
-          cards={solutions.cards}
           ctas={solutions.ctas}
-        />
+        >
+          {solutions.cards.map((card, index) => {
+            return (
+              <CardSliderElement key={index}>
+                <CardColored
+                  color={card.color}
+                  title={card.title}
+                  description={card.description}
+                />
+              </CardSliderElement>
+            )
+          })}
+        </CardSlider>
       </Container>
 
-      <Container background="stone" padding="both-padding">
+      <Container id="Projects" background="stone" padding="both-padding">
```

Comment:
Just a question, why did you add id's on the containers? Are they utilized somewhere?

Reply:
Nope, not yet, just for organising it a bit :-)